templates/master/00-master/baremetal/files/baremetal-haproxy.yaml

```diff
@@ -11,7 +11,7 @@ contents:
       creationTimestamp:
       deletionGracePeriodSeconds: 65
       labels:
-        app: openstack-infra-api-lb
+        app: kni-infra-api-lb
```

Comment:
Makes sense.

Reply:
Do we want to expose the KNI name here? We may want to call this baremetal instead.

Reply:
I suppose the other things are already named KNI in MAO.  It's confusing but at least this makes it consistent.